src/products/index.ts

```diff
@@ -26,14 +26,16 @@ class Products {
     ({ ...ProductTransformer.transformer(this.structure)(data), meta })
 
   init() {
+    this.updateProducts();
```

Comment:
Could you leave a comment here to explain why you do this? It will be easier for future referencec :)